game.js

```diff
@@ -0,0 +1,148 @@
+let bot = require("./bot.js");
+let helpers = require("./helpers.js");
+let lastLetter = "";
+let cities = [
+  "Архангельск",
+  "Борисполь",
+  "Воронеж",
+  "Гюмри",
+  "Донецк",
+  "Енакиево",
+  "Жуковский",
+  "Запорожье",
+  "Изюм",
+  "Йошкар-Ола",
+  "Киев",
+  "Лондон",
+  "Москва",
+  "Николаев",
+  "Орёл",
+  "Париж",
+  "Ростов",
+  "Селидово",
+  "Трускавец",
+  "Уфа",
+  "Флоренция",
+  "Харьков",
+  "Цюрих",
+  "Челябинск",
+  "Шанхай",
+  "Щёлково",
+  "Ыгдыр",
+  "Электросталь",
+  "Юрмала",
+  "Ялта"
+].map(item => item.toLowerCase());
+
+exports.session = class Session {
```

Comment:
No need in class right here. Use just an object

Reply:
```suggestion
let sessions = {
  makeSession(chatID) {
    this[chatID] = { spentCities: new Set() };
  },
  deleteSession(chatID) {
    delete this[chatID];
  }
};
```
ready